packages/utils/src/ts-eslint/ESLint.ts

```diff
@@ -1,11 +1,11 @@
 /* eslint-disable @typescript-eslint/no-namespace */
 
 export {
-  // TODO - remove this in the next major
+  // TODO(v9) - remove this in the next major
```

Comment:
better comment would be to remove this when eslint no longer exports it (which will likely be their next major)